tests/test_plotter.py

```diff
@@ -170,7 +170,8 @@ def test_plotter_remains_shallow():
     assert np.shares_memory(sphere['numbers'], sphere2['numbers'])
 
 
-def test_add_multiple(sphere):
+def test_add_multiple(sphere, verify_image_cache):
```

Comment:
Since `verify_cache_image` is a regular fixture,i.e. not autoused, why is it needed to use it in the function signature and then skip it?

Reply:
Totally right about the wrapper. 

I missed this file when I removed the autouse flag in the plugin, should be removed.